bsuite/environments/mdp_playground.py

```diff
@@ -0,0 +1,108 @@
+# pylint: disable=g-bad-file-header
+# Copyright 2019 .... All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or  implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""The MDP Playground reinforcement learning environment."""
+
+# Import all packages
+
+from mdp_playground.envs import RLToyEnv #mdp_playground
+
+# import collections
+from bsuite.experiments.mdp_playground import sweep
+from bsuite.environments import base
+from bsuite.utils.gym_wrapper import DMEnvFromGym, space2spec
+import dm_env
+from dm_env import specs
+from dm_env import StepType
+import gym
+import numpy as np
+from typing import Any
+
+# def ohe_observation(obs):
+
+class DM_RLToyEnv(base.Environment):
+  """A wrapper to convert an RLToyEnv Gym environment from MDP Playground to a
+  base.Environment which is a subclass of dm_env.Environment.
+  Based on the DMEnvFromGym in gym_wrapper.py"""
+
+  def __init__(self, max_episode_len: int = 100, **config: Any):
```

Comment:
Could you rename `config` to something like `gym_make_kwargs`? Ideally the type would be something like `Mapping[str, Any]` too.